templates/gig/gig_detail.html

```diff
@@ -324,7 +325,7 @@
 {% endif %}
 
 $(document).ready(function() {
-    init_plan_buttons("{{csrf_token}}")
+    // init_plan_buttons("{{csrf_token}}")
```

Comment:
I'd just delete these, rather than commenting them out.  If we need them again, they're in the git history.  Otherwise, we're going to wonder if these were important when we notice the comments six months from now.

Reply:
Yes - the javascript has always been messy and now that we're using django to handle some of the form validation, and now htmx, it all needs going over with a fine-tooth comb.